bioimageio/spec/shared/utils.py

```diff
@@ -182,8 +181,10 @@ def transform_ImportableModule(node):
 
     @staticmethod
     def transform_ResolvedImportableSourceFile(node: ResolvedImportableSourceFile) -> nodes.ImportedSource:
+        module_path = resolve_uri(node.source_file)
+        module_name = os.path.splitext(os.path.split(module_path)[1])[0]
```

Comment:
as we predominantly use pathlib, I'd suggest
```suggestion
        module_name = module_path.stem
```
however I don't see why using `module_name` is a good idea here... (see comment below)